crates/fluvio-cluster/src/delete.rs

```diff
@@ -149,16 +150,23 @@ impl ClusterUninstaller {
         pb.set_message("Uninstalling fluvio local components");
 
         let kill_proc = |name: &str, command_args: Option<&[String]>| {
+            sysinfo::set_open_files_limit(0);
             let mut sys = System::new();
-            sys.refresh_processes(); // Only load what we need.
-            for process in sys.processes_by_exact_name(name) {
+            sys.refresh_processes(sysinfo::ProcessesToUpdate::All); // Only load what we need.
```

Comment:
this seems like repeating. can you change into common util?

Reply:
Sure! I created a `process` mod.